docs/web-console-guide/master.adoc

```diff
@@ -25,17 +25,17 @@ include::topics/mta-what-is-the-toolkit.adoc[leveloffset=+2]
 // About the {WebName}
 include::topics/about-the-user-interface.adoc[leveloffset=+2]]
 
-[id=mta-6-ui-interface-views]
+[id=mta-ui-interface-views]
 == User interface views
 
 The {ProductName} ({ProductShortName}) {WebName} has two views:
 
 * Administration view
 * Migration view
 
-In *Administration* view, you configure the instance environment, working with credentials, repositories, HTTP and HTTPS proxy definitions, custom migration targets, and issue management.
+In *Administration* view, administrators can configure the instance environment, work with credentials and repositories, define HTTP and HTTPS proxies, specify custom migration targets, manage issues, and add custom assessment questionnaires.
```

Comment:
I noticed one minor issue, otherwise the PR LGTM  .

Replace 'specify custom migration targets' 
with
'create custom migration targets'.